### ISSUE TITLE
Scrolling is slow on simulator/device

### ISSUE THREAD

**Issue by marco-c** [andreasgal/j2me.js#845]
We're probably doing something wrong with gestures, because the simulator should be as fast as desktop.

**Comment by marco-c** [comment]
We might need to properly implement the FrameAnimator class for this to feel snappy.

**Comment by mykmelez** [comment]
Indeed, I see warnings about two unimplemented *FrameAnimator* methods when I try to scroll a long list in a midlet on an Aries device (which is about as fast as an FxOS device can get):

>com/nokia/mid/ui/frameanimator/FrameAnimator.kineticScroll.(IIIF)V not implemented
>com/nokia/mid/ui/frameanimator/FrameAnimator.stop.()V not implemented

Unsure if it's related, but I also see this message about *GestureInteractiveZone.isSupported* around the same time:

> GestureInteractiveZone.isSupported.(I)Z not implemented (64)
